ui/tests/e2e/service-mesh-performance.spec.js

```diff
@@ -0,0 +1,90 @@
+import { test, expect } from '@playwright/test';
+import { ENV } from './env';
+
+test.describe('Service Mesh Performance Management Tests', () => {
```

Comment:
Can we have `{ tag: '@unstable' }` here as well to make sure the test is stable first?